package.json

```diff
@@ -3,7 +3,15 @@
   "license": "MIT",
   "devDependencies": {
     "async": "^1.5.0",
-    "babel": "5.8.21",
+    "babel-core": "^6.3.17",
+    "babel-plugin-add-module-exports": "0.1.2",
+    "babel-plugin-transform-runtime": "^6.3.13",
+    "babel-plugin-transform-class-properties": "^6.6.0",
+    "babel-plugin-transform-flow-strip-types": "^6.3.13",
+    "babel-runtime": "^6.0.0",
+    "babel-preset-es2015": "^6.6.0",
+    "babel-preset-es2015-loose": "^7.0.0",
```

Comment:
Options: use loose preset, add each plugin and specify loose mode for each, or land something like https://github.com/babel/babel/pull/3331